hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/HawkbitCommonUtil.java

```diff
@@ -1351,23 +1351,23 @@ private static boolean isNoTargets(final Long... statusCount) {
     private static Long getStatusCount(final String propertName, final Item item) {
         return (Long) item.getItemProperty(propertName).getValue();
     }
-
+    
     /**
-     * Get the formatted string of status and target count details.
+     * Get the formatted string of status and target name.
      * 
      * @param details
-     *            details of status and count
-     * @return String
+     *            details of status and name
+     * @return
      */
-    public static String getFormattedString(Map<Status, Long> details) {
+    public static String getNameStatusFormattedString(Map<String, String> details) {
         StringBuilder val = new StringBuilder();
-        if (details == null || details.isEmpty()) {
-            return null;
-        }
-        for (Entry<Status, Long> entry : details.entrySet()) {
-            val.append(entry.getKey()).append(":").append(entry.getValue()).append(",");
+        if (details != null && !details.isEmpty()) {
```

Comment:
Yes that's okay but use early return for the if statement.
see http://programmers.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement